test/performance/README.md

```diff
@@ -0,0 +1,82 @@
+# Kueue Performance Testing
+
+## Measurements
+
+### Job startup latency
+
+How fast do jobs transition from `created` to `started` state?
+Time spent between the transition from `job.CreationTimestamp.Time` to `job.Status.StartTime.Time` state.
+
+High Job startup latency in Kueue is expected when the total quota is not enough to schedule all jobs immediately, because the jobs need to queue.
+
+### Job startup throughput
+
+The best workload admission rate per second within 1 minute intervals. 
+The rate is measured every 5 seconds (see more details in [PromQL examples](https://prometheus.io/docs/prometheus/latest/querying/examples/#subquery)):
+
+`max_over_time(sum(rate(kueue_admitted_workloads_total{cluster_queue="{{$clusterQueue}}"}[1m]))[{{$testTimeout}}:5s])`
+
+This measurement is not efficient if the cluster quota is big enough to schedule all workloads of the test immediately, because Kueue immediately admits all the workloads and the `kueue_admitted_workloads_total` never increases. In this case, the PromQL query returns 0.
```

Comment:
```suggestion
This measurement is not accurate if the cluster quota is big enough to schedule all workloads of the test immediately, because Kueue immediately admits all the workloads and the `kueue_admitted_workloads_total` never increases. In this case, the PromQL query returns 0.
```